src/renderers/shared/stack/event/EventConstants.js

```diff
@@ -87,7 +87,9 @@ var topLevelTypes = keyMirror({
   topVolumeChange: null,
   topWaiting: null,
   topWheel: null,
-});
+};
+
+export type TopLevelTypes = $Enum<typeof topLevelTypes>;
```

Comment:
Is this anywhere in Flow documentation? It’s a bit odd that powerful/useful features are undocumented.